Formula/youtubeuploader.rb

```diff
@@ -0,0 +1,54 @@
+class Youtubeuploader < Formula
+  desc "Scripted uploads to Youtube"
+  homepage "https://github.com/porjo/youtubeuploader"
+  url "https://github.com/porjo/youtubeuploader/archive/21.05.tar.gz"
+  sha256 "cc087e05e9a31408a6941030cdb933e6e52d619960e765dbb3d91e3661d3dc98"
+  license "Apache-2.0"
+  head "https://github.com/porjo/youtubeuploader.git", branch: "master"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    path = buildpath/"src/github.com/porjo/youtubeuploader"
+    path.install Dir["*"]
+    cd path do
+      system "go", "build", "-a", "-ldflags", "-s -X main.appVersion=#{version}", "-o", "#{bin}/youtubeuploader"
+      prefix.install_metafiles
+    end
```

Comment:
```suggestion
    system "go", "build", "-a", *std_go_args(ldflags: "-s -X main.appVersion=#{version}")
```
Does this not work? We don't want to have to set `GOPATH`, as this is no longer needed for building up-to-date Go formulae. What does the `-a` flag do?

Reply:
Yes, this works perfectly too, also without the `-a` flag.

The suggested changes have just been pushed.